docs/source/configuration/credentials.md

```diff
@@ -15,7 +15,7 @@ The following examples all use the default `ConfigLoader` class.
 from kedro.config import ConfigLoader
 from kedro.framework.project import settings
 
-conf_path = str(project_path / settings.CONF_SOURCE)
+conf_path = f"{project_path}/{settings.CONF_SOURCE}"
```

Comment:
Now that I look at this, where does `project_patg` come from?

Is the `TypeError` because you filled a string and this was expecting a `pathlib.Path` ?

Reply:
In my case I'm using project_path = os.getcwd() so it is a str

Reply:
> Now that I look at this, where does `project_patg` come from?
> 
> Is the `TypeError` because you filled a string and this was expecting a `pathlib.Path` ?

ohh ok! I thought that project_path was a string, so you say it should be a pathlib.Path object?

Reply:
Correct! But the fact that you got it wrong is a documentation bug 😁 could you revert these changes and add a line at the beginning that assigns project_path as a Path object?

Reply:
Hi @jmnunezd -- would you be able to make the change suggested so I can approve and merge please?

Reply:
so it will be adding to the docs something like:

```
from pathlib import Path
project_path = Path(my_project_path)

conf_path = str(project_path / settings.CONF_SOURCE)

...
```

right?

Reply:
Correct @jmnunezd! Please give it a try just to verify if it works. It should be a matter of adding the first 2 lines, as I understand that the third one was already there.

Reply:
Done! It's now ready for a new review

Reply:
I imported Path before kedro to make it compliant with [PEP8 import's order ](https://peps.python.org/pep-0008/#imports)

```
from pathlib import Path

from kedro.config import ConfigLoader
from kedro.framework.project import settings
```